.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["es2015", "react", "stage-0"],
-  "plugins": ["add-module-exports"]
+  "plugins": ["add-module-exports", "transform-flow-strip-types"]
```

Comment:
@okonet By default, the preset [react](https://babeljs.io/docs/plugins/preset-react/) includes the `transform-flow-strip-types` plugin. And right now it is not possible to disable a specific plugin from preset, AFAIK.
[http://stackoverflow.com/questions/33707189/is-there-an-option-to-disable-a-plugin-in-babel-when-using-a-preset](http://stackoverflow.com/questions/33707189/is-there-an-option-to-disable-a-plugin-in-babel-when-using-a-preset)